errors/stack/stack.go

```diff
@@ -19,12 +19,12 @@ var (
 )
 
 type Frame struct {
-	Filename string `json:"filename"`
-	Method   string `json:"method"`
-	Line     int    `json:"lineno"`
+	Filename string
+	Method   string
+	Line     int
```

Comment:
Why do we no longer need the json annotations? I think (not 100% sure) the RPC endpoint uses these annotations because it returns the json-marshaled error